torch/_dynamo/variables/builder.py

```diff
@@ -1497,7 +1497,7 @@ def wrap_tensor(self, value: torch.Tensor):
 
         self.tx.output.input_source_to_var[source] = tensor_variable
         assert "tensor_dict" not in tensor_proxy.node.meta
-        tensor_proxy.node.meta["tensor_dict"] = value.__dict__.copy()
+        tensor_proxy.node.meta["tensor_dict"] = {k: None if isinstance(v, torch.Tensor) else v for k, v in value.__dict__.items()}
```

Comment:
is a similar change needed here?
https://github.com/pytorch/pytorch/blob/f0da167ce55ddbbe4f0138ed0f0804008069ddad/torch/_dynamo/output_graph.py#L810

Reply:
oh and here:

https://github.com/pytorch/pytorch/blob/6b28af1b79eaa63e2f423d925bbd42330582983f/torch/_dynamo/variables/builder.py#L1141-L1143